fig_test.go

```diff
@@ -259,6 +270,7 @@ func Test_fig_Load_Defaults(t *testing.T) {
 				want.Logger.Production = false
 				want.Logger.Metadata.Keys = []string{"ts"}
 				want.Application.BuildDate = time.Date(2020, 1, 1, 12, 0, 0, 0, time.UTC)
+				want.Server.Listener = ListenerTCP
```

Comment:
This should be `ListenerUnix` based on the default.

Reply:
My bad! The value is actually correct, but I forgot to commit the updated `testdata/server.*` files, which hold `tcp` as value. I wanted to make sure that default and actual value are different. Will be commited in a few.